src/components/Search/Input.tsx

```diff
@@ -57,17 +57,31 @@ const SearchSlash = styled.p`
   }
 `
 
-const Input = ({ query, setQuery, refine, inputRef, ...rest }) => {
+interface IInputProps {
+  query: string
+  setQuery: (arg: string) => void
+  refine: (arg: string) => void
```

Comment:
To be more explicit and clear
```suggestion
  refine: (query: string) => void
```